utils/src/main/kotlin/Extensions.kt

```diff
@@ -237,7 +237,8 @@ fun String.encodeOrUnknown() = fileSystemEncode().takeUnless { it.isEmpty() } ?:
 
 /**
  * Return the string encoded for safe use as a file name. Also limit the length to 255 characters which is the maximum
- * length in most modern filesystems: https://en.wikipedia.org/wiki/Comparison_of_file_systems#Limits
+ * length in most modern filesystems, see
+ * [comparison of file system limits](https://en.wikipedia.org/wiki/Comparison_of_file_systems#Limits).
```

Comment:
Nit: I though the logic when you use `[1]`-style references is if the URL is too long to fit the line, like here.

Reply:
I used it only if the line containing only the link gets longer than 120 characters so far.